package.json

```diff
@@ -48,13 +49,14 @@
     "request": "2.65.0",
     "scrypt-hash": "1.1.13",
     "through": "2.3.8",
-    "uuid": "1.4.1"
+    "uuid": "1.4.1",
+    "web-push": "git+https://github.com/eoger/web-push.git#node-0.10"
```

Comment:
We were talking about this with @ckarlof 10 min ago, I think I can manage to make a version that uses polyfills only when necessary to get my changes accepted upstream